charts/k8s-service/values.yaml

```diff
@@ -372,6 +372,27 @@ additionalContainerEnv: {}
 #         envVarName: CONFIG_FOO
 configMaps: {}
 
+# persistentVolumes is a list of maps that specifies PeristantVolumes that should be mounted on the pod.
+# Each entry represents a persistent volume which should already exist within your cluster. The value is also a map and has the following attributes:
+#   - volumeName (string) (required)
+#     : The logical name of the Persistent Volume, as it will be registered in kubernetes.
+#   - mountPath (string) (required)
+#     : The path within the container upon which this volume should be mounted.
+#   - claimName (string) (required)
+#     : The name of the Persistent Volume Claim on which this Persistent Volume in bound.
+#
+# EXAMPLE:
+# persistentVolumes:
+# - volumeName: example-pv
+#   mountPath: /mnt/myVol
+#   claimName: example-pv-claim
+# - volumeName: example-pv-2:
+#   mountPath: /mnt/myOtherVol
+#   claimName: example-pv2-claim
+#
+#
+persistentVolumes: []
```

Comment:
It is better to organize this as a map, where the key is the volume name. The primary reason is that it makes it easier to override with `--set` if needed.